dist/stat_data.py

```diff
@@ -402,6 +402,7 @@ def __init__(self, name, desc, flags=''):
     CheckpointStat('checkpoint_time_min', 'min time (msecs)', 'no_clear,no_scale'),
     CheckpointStat('checkpoint_time_recent', 'most recent time (msecs)', 'no_clear,no_scale'),
     CheckpointStat('checkpoint_time_total', 'total time (msecs)', 'no_clear,no_scale'),
+    CheckpointStat('checkpoint_tree_duration', 'transaction checkpoint tree duration (usecs)'),
```

Comment:
Sorry to be so picky, but this section does not need the `transaction checkpoint` prefix at the start of the description string. It's also not pluralised like I mentioned before - how about we reword this, what do you think about "time spent on per-tree checkpoint work"?

Reply:
@korteland 
Would it be better to use “time spent on all-tree checkpoint work”, __checkpoint_apply_to_dhandles traverse all btree handle?

What's your opinion？

Reply:
@y123456yz I think "per-tree" is better - "all-tree" isn't an idiomatic way to express that. And it could be mistaken for work that would apply to every tree at once (e.g. updating metadata), rather than an accumulated amount of per-tree work.

Reply:
@korteland 
hi, @korteland 
I have change the code, please review again.
thanks
:)